x-pack/plugins/security_solution/public/detection_engine/rule_exceptions/components/add_exception_flyout/index.tsx

```diff
@@ -520,9 +533,10 @@ export const AddExceptionFlyout = memo(function AddExceptionFlyout({
           <ExceptionItemComments
             accordionTitle={
               <SectionHeader size="xs">
-                <h3>{i18n.COMMENTS_SECTION_TITLE(0)}</h3>
+                <h3>{i18n.COMMENTS_SECTION_TITLE(newComment ? 1 : 0)}</h3>
               </SectionHeader>
             }
+            initialIsOpen={!!newComment} // TODO: Ask yara if this the expected behavior
```

Comment:
👋🏽 😄 Are you wondering if this is the behavior I suggested of adding the comment prepopulated with alert id? I think this works!

Reply:
Yes 😅 the behavior of adding the comment as well as opening the panel by default.

Reply:
I think that looks good as you have it! Just whatever product/docs suggests as far as copy.